Lombiq.UIKit.Widgets/Views/Widget-CarouselWidget.cshtml

```diff
@@ -0,0 +1,35 @@
+@using Lombiq.UIKit.Widgets.Models
+@using Lombiq.UIKit.Widgets.Settings
+@using OrchardCore.ContentManagement
+@using OrchardCore.ContentManagement.Metadata
+@using OrchardCore.DisplayManagement
+@using OrchardCore.Flows.Models
+
+@inject IContentDefinitionManager _contentDefinitionManager;
+
+@{
+    var typeDefinition = await _contentDefinitionManager.GetTypeDefinitionAsync("CarouselWidget");
+    var part = typeDefinition.Parts.Where(part => part.Name == nameof(CarouselWidgetPart)).FirstOrDefault();
+    CarouselWidgetPartSettings settings = new CarouselWidgetPartSettings();
+    if(part != null)
+    {
+        settings = part.GetSettings<CarouselWidgetPartSettings>();
+    }
+}
+
+<div class="carouselWidget" id="carouselWidget-@Model.ContentItem.ContentItemId">
+    @{
+        var bagItems = ((ContentItem)Model.ContentItem).As<BagPart>().ContentItems.Select(contentItem => contentItem.As<SlidePart>());
+        var items = new List<IShape>();
+
+        bagItems.ForEach(async (slide) => items.Add(await New.Slide(Title: slide.Title.Text, Image: slide.Image.Paths[0])));
+
+        var carouselSettings = settings.Options;
+    }
+
+    <shape type="SlickCarousel"
+           AdditionalClasses="slickCarouselSample"
```

Comment:
Why adding this class with `sample` text? From what I understand it's a final shape for the widget so it shouldn't be a sample but the actually shape. Maybe you wanted to add `carouselWidget__widget`?